test/common/listener_manager/listener_manager_impl_test.cc

```diff
@@ -400,6 +400,95 @@ name: bar
   addOrUpdateListener(parseListenerFromV3Yaml(yaml2));
 }
 
+TEST_P(ListenerManagerImplWithRealFiltersTest, AllowAddressesUpdateParitially) {
+  const std::string yaml1 = R"EOF(
+name: foo
```

Comment:
Please add a comment saying what part of the listener config is changing so the reader doesn't have to manually diff the two configs in their head.